plugin/s3/plugin.go

```diff
@@ -142,12 +141,10 @@ func (p S3Plugin) Validate(endpoint plugin.ShieldEndpoint) error {
 		ansi.Printf("@G{\u2713 access_key_id}        @C{%s}\n", s)
 	}
 
-	s, err = endpoint.StringValueDefault("s3_port","443")
+	s, err = endpoint.StringValueDefault("s3_port", "")
```

Comment:
Please validate (somewhere in here) that s3_host is specified if s3_port is given.